client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/RemoteCacheImpl.java

```diff
@@ -344,6 +344,12 @@ public V put(K key, V value, long lifespan, TimeUnit lifespanUnit, long maxIdleT
       return await(putAsync(key, value, lifespan, lifespanUnit, maxIdleTime, maxIdleTimeUnit));
    }
 
+   public V putDuringStart(K key, V value) {
```

Comment:
Why do we need this method exactly?

Reply:
Because we perform the operation before `start = true` and all `RemoteCacheImpl` operations call `assertRemoteCacheManagerIsStarted()` which will throw an exception in this case. Of course I could just call `.put` after we set `start = true`, however I don't think that is correct as this operation is now part of the `RemoteCacheManager` starting and operations should not proceed until it has completed startup.